plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -152,9 +154,16 @@ public OpenShiftConnector(DockerConnectorConfiguration connectorConfiguration,
         this.openShiftLivenessProbeDelay = openShiftLivenessProbeDelay;
         this.openShiftLivenessProbeTimeout = openShiftLivenessProbeTimeout;
 
-        Config config = new ConfigBuilder().withMasterUrl(openShiftApiEndpoint)
-                .withUsername(openShiftUserName)
-                .withPassword(openShiftUserPassword).build();
+        Config config;
+        if (StringUtils.isNotBlank(openShiftToken)) {
```

Comment:
usually che code is using https://google.github.io/guava/releases/19.0/api/docs/com/google/common/base/Strings.html#isNullOrEmpty(java.lang.String)